### PR TITLE
Create somefile.md

### PR DESCRIPTION
**Pull request by oggius**
Created somefile

### CODE REVIEW

**Comment by oggius3** [comment]
Why is this pull request required?

File: somefile.md [hunk 1]
```diff
@@ -0,0 +1 @@
+somefile
```
Comment (oggius3) [inline somefile.md]: What is the reason of creating this file?

**Review by oggius3 — COMMENTED** [review]
Simple comment feedback

File: somefile.md (hunk 1 above)
Comment (oggius) [reply somefile.md]: This file is required, just in case
```suggestion
somefile2.md
```

Comment (oggius3) [inline somefile.md]: Rename this file to somefile2.md

**Comment by oggius3** [comment]
Add one more file, named newfile3.md

File: somefile.md (hunk 1 above)
Comment (oggius3) [reply somefile.md]: and also, add content with value 'testme'

**Review by oggius3 — CHANGES_REQUESTED** [review]

**Review by oggius3 — APPROVED** [review]

**Pull request merged**